templates/education/dnso-profiles.php

```diff
@@ -0,0 +1,143 @@
+$clone_name_title_room_phone_email = ( isset( $profile_cct['clone_name_title_room_phone_email']['textarea'] ) ) ? $profile_cct['clone_name_title_room_phone_email']['textarea'] : false;
+$phone1              = ( isset( $profile_cct['phone']['0']['tel-1'] ) ) ? $profile_cct['phone']['0']['tel-1']  : false;
+$phone2              = ( isset( $profile_cct['phone']['0']['tel-2'] ) ) ? $profile_cct['phone']['0']['tel-2']: false;
+$phone3              = ( isset( $profile_cct['phone']['0']['tel-3'] ) ) ? $profile_cct['phone']['0']['tel-3']: false;
+$email               = ( isset( $profile_cct['email']['0']['email'] ) ) ? $profile_cct['email']['0']['email'] : false;
+$clone_bio           = ( isset( $profile_cct['clone_bio']['textarea'] ) ) ? $profile_cct['clone_bio']['textarea'] : false;
+$clone_office        = ( isset( $profile_cct['clone_office']['0']['text'] ) ) ? $profile_cct['clone_office']['0']['text'] : false;
+
+?>
+
+<div id="<?php echo absint( $post_id ); ?>" class="<?php echo esc_attr( $plain_term_slug ); ?> iso-profile">
+
+  <div class="boxey <?php echo esc_attr( $odd_even ); ?> <?php echo esc_attr( $plain_term_slug ); ?> profile">
+
+    <div class="boxey-inside">
+
+        <div class="boxey-inner">
+
+          <div class="iso-featured-image"><a href="#modal_<?php echo absint( $post_id ); ?>" class="fancybox-inline"><img src="<?php echo esc_url( $post_thumbnail ); ?>" alt="" /></a></div>
+
+          <h3><div class="iso-title">
+
+            <a href="#modal_<?php echo absint( $post_id ); ?>" class="fancybox-inline">
+            <?php if ( false !== $salutations ) : echo esc_html( $salutations ); endif; ?> <?php if ( false !== $first_name ) : echo esc_html( $first_name ); endif; ?> <?php if ( false !== $middle ) : echo esc_html( $middle ); endif; ?> <?php if ( false !== $last_name ) : echo esc_html( $last_name ); endif; ?>          
+            <small><?php if ( false !== $position ) : echo esc_html( $position ); endif; ?></small></a>
+
+          </div></h3>
+
+          <div class="iso-description hidden">
+            <?php if ( false !== $salutations ) : echo esc_html( $salutations ); endif; ?> 
+            <?php if ( false !== $first_name ) : echo esc_html( $first_name ); endif; ?> 
+            <?php if ( false !== $middle ) : echo esc_html( $middle ); endif; ?> 
+            <?php if ( false !== $last_name ) : echo esc_html( $last_name ); endif; ?>
+            <?php if ( false !== $position ) : echo esc_html( $position ); endif; ?> 
+            <?php if ( false !== $clone_name_title_room_phone_email ) : echo esc_html( $clone_name_title_room_phone_email ); endif; ?> 
+            <?php echo esc_html( $plain_term_name ); ?>
+          </div>
+
+          <?php if ( false !== $clone_room_number ) : echo esc_html( $clone_room_number ); endif; ?></br>
+
+          <?php if ( false !== $email ) :  echo wp_kses_post( "\n"."<i class='icon-envelope'></i> <a href='mailto:$email'>$email</a>"."\n" ) ;  endif; ?></a></br>
```

Comment:
Same comment as in https://github.com/ubc/ubc-use-template-shortcode/pull/32 - let's try and take markup out of PHP